### PR TITLE
feat(system-e2e): Add e2e tests for session history

### PR DESCRIPTION
**Pull request by saevarma**
# Add E2E tests for session history

[Notion task](https://www.notion.so/Write-E2E-tests-9adac249722044f682ca7467fc33e810)

## What

Adding E2E test cases for session history:
* Verifies that the table loads with at least one record.
*  Verifies that the table can be filtered by nationalId.

## Why

To add automatic E2E test for sessions.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/system-e2e/src/tests/islandis/service-portal/smoke/sessions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    const page = await context.newPage()
+
+    // Act
+    const sessionsRows = page.locator('table > tbody > tr')
+    await page.goto(sessionHistoryUrl, {
+      waitUntil: 'networkidle',
+    })
+
+    // Assert
+    await expect(sessionsRows).toHaveCountGreaterThan(0)
```
Comment (krrrrrrrrrrr) [inline apps/system-e2e/src/tests/islandis/service-portal/smoke/sessions.spec.ts:38]: I'm expecting sessionRows to be detached on `goto`

**Review by krrrrrrrrrrr — COMMENTED** [review]
Great tests :)

Many selectors could be rewritten to use `page.getBy(Role|Label)`

File: apps/system-e2e/src/tests/islandis/service-portal/smoke/sessions.spec.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    const page = await context.newPage()
+    await page.goto(homeUrl, {
+      waitUntil: 'domcontentloaded',
+    })
+    await page.locator('data-testid=user-menu >> visible=true').click()
+    await page.locator('role=button[name="Skipta um notanda"]').click()
+    await page.locator('role=button[name*="ARTIC ehf."]').click()
+    await page.waitForURL(new RegExp(homeUrl), {
+      waitUntil: 'domcontentloaded',
+    })
```
[the remark this answers is not in the document]
Comment (saevarma) [reply apps/system-e2e/src/tests/islandis/service-portal/smoke/sessions.spec.ts]: Indeed it does, thanks!

File: apps/system-e2e/src/tests/islandis/service-portal/smoke/sessions.spec.ts (hunk 1 above)
Comment (saevarma) [reply apps/system-e2e/src/tests/islandis/service-portal/smoke/sessions.spec.ts]: Yes think a reorder between arrange and act confused the order. But for some strange reasons this actually works. But I reversed the order to be more logical.

**Review by krrrrrrrrrrr — APPROVED** [review]

**Pull request merged**